src/sdk/sql_cluster_router.cc

```diff
@@ -2951,7 +2951,50 @@ hybridse::sdk::Status SQLClusterRouter::HandleDeploy(const hybridse::node::Deplo
                 }
             }
             if (index_id_set.count(openmldb::schema::IndexUtil::GetIDStr(column_key)) > 0) {
-                // skip exist index
+                // if type match && new ttl greater than old ttl, update ttl, else skip exist index
+                ::openmldb::common::ColumnKey old_column_key;
+                for (const auto& table_column_key : it->second.column_key()) {
+                    if (openmldb::schema::IndexUtil::GetIDStr(table_column_key) ==
+                        openmldb::schema::IndexUtil::GetIDStr(column_key)) {
+                        old_column_key = table_column_key;
+                        break;
+                    }
+                }
+                if (old_column_key.ttl().ttl_type() != column_key.ttl().ttl_type()) {
+                    return {::hybridse::common::StatusCode::kCmdError,
```

Comment:
two indexes will be consider as the same index if the column names and ts are same(see `GetIDStr()`). so we can't add a new 'c1|ts|lat ttl'.   Return because new index ttl type mismatch old index ttl type when update the same index

Reply:
Why are they the same? If you don't want two indexes, you should merge them to one, abs or lat